### ISSUE TITLE
No Mapbox page shell

### ISSUE THREAD

**Issue by davidtheclark** [mapbox/android-docs#148]
@mayagao @cammace: Are you opposed to adding the Mapbox page shell to this site (header, logo, footer, navigation)? It's not there now but seems to me that it should be — was that just a technical limitation that you hoped would be solved with a future iteration (and will be!)?

**Comment by davidtheclark** [comment]
Adding the page shell will introduce a few other complications:

- The current mobile header will no longer work, because it has its own navigation that would conflict with the Mapbox navigation.
- The way the current header is fixed to the top of the screen will have to be readdressed — either unfixing it (my preference) or only fixing after you've scrolled beyond a certain point.

**Comment by mayagao** [comment]
I actually want a simpler custom header for all developer related pages (documentation, help, etc)

<img width="1034" alt="screen shot 2017-07-26 at 11 25 43 am" src="https://user-images.githubusercontent.com/5186564/28629358-31bce32e-71f5-11e7-8755-0f30eb8d06a7.png">

I don't think we need the footer for documentations.

**Comment by cammace** [comment]
for the mobile header, I can work on resolving this later on. I'd prefer to keep the menu fixed after you've scrolled beyond a certain point.

**Comment by davidtheclark** [comment]
@mayagao: How would you feel about the regular header for now? — because creating a custom header for these pages that includes similar functionality will be a lot of work.

**Comment by davidtheclark** [comment]
Actually, thinking again —— maybe not too much work if we make a React component for it and dictate that all the pages with this header will have to use the React component.

**Comment by mayagao** [comment]
@davidtheclark that sounds good for now but I don't think that can be the long term solution.

When user is inside the documentation, the most important thing is help them navigate inside that documentation (or go to related doc pages), not go to other parts of mapbox.com. Our docs are scattered around everywhere, which makes the nav menu design really challenging. Keeping the default header is going to make it even more difficult because that is going to introduce one more level of hierarchy:

1. mapbox.com pages
2. documentation pages (ios, unity, android,help, api...)
3. pages inside one documentation (examples, main doc ...)
4. sections inside one page (overview, install ...)

I did some rough sketches when designing the unity doc:

<img width="697" alt="screen shot 2017-07-26 at 11 35 41 am" src="https://user-images.githubusercontent.com/5186564/28630143-75ef54d0-71f7-11e7-8812-d0f654a75213.png">

Basically remove 1, put 2 and 3 to the top, and then move 4 to the bottom. This design looks pretty overwhelming and obviously needs more work (like if we want to add search ...).

**Comment by mgosline** [comment]
@mayagao I'm working on incorporating a top nav into studio that pairs with the mapbox.com & signed in/dashboard nav. Can we sync up so that the developer related navs align with the rest of the navs in terms of IA and style?

**Comment by langsmith** [comment]
@davidtheclark @colleenmcginnis @mgosline , does the new docs design resolve this ticket?

**Comment by davidtheclark** [comment]
Yep!

![it-is-done](https://user-images.githubusercontent.com/628431/43029942-ce265708-8c3f-11e8-9966-79fc8583ccc7.gif)
